include/openssl/symhacks.h

```diff
@@ -34,4 +34,4 @@
 
 # endif
 
-#endif                          /* ! defined HEADER_VMS_IDHACKS_H */
+#endif                          /* ! defined OPENSSL_VMS_IDHACKS_H */
```

Comment:
this comment should name OPENSSL_SYMHACKS_H,
can anybody explain why there is a internal/symhacks.h and a openssl/symhacks.h at the same time?

Reply:
> can anybody explain why there is a internal/symhacks.h and a openssl/symhacks.h at the same time?

One concerns internal symbols while the other concerns public symbols.

Reply:
ah, okay, thanks.